From 208a5cae51f248f98684463f51842fdeb6de6624 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 14 Dec 2011 11:34:17 +0800
Subject: [PATCH 0738/2463] ENGR00170225 mx5x: build failed due to ipuv3 fb
 platform data change

The change impact the mx5 bbg and loco build.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 arch/arm/mach-mx5/board-mx51_babbage.c |    4 ++--
 arch/arm/mach-mx5/board-mx53_loco.c    |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx5/board-mx51_babbage.c b/arch/arm/mach-mx5/board-mx51_babbage.c
index b808dfd..f8661ff 100755
--- a/arch/arm/mach-mx5/board-mx51_babbage.c
+++ b/arch/arm/mach-mx5/board-mx51_babbage.c
@@ -614,9 +614,9 @@ static void __init fixup_mxc_board(struct machine_desc *desc, struct tag *tags,
 		gpu_data.reserved_mem_size = gpu_mem;
 
 		/* reserver memory for fb */
-		bbg_fb_data[0].res_base = gpu_data.reserved_mem_base
+		bbg_fb_data[0].res_base[0] = gpu_data.reserved_mem_base
 					+ gpu_data.reserved_mem_size;
-		bbg_fb_data[0].res_size = fb_mem;
+		bbg_fb_data[0].res_size[0] = fb_mem;
 	}
 }
 
diff --git a/arch/arm/mach-mx5/board-mx53_loco.c b/arch/arm/mach-mx5/board-mx53_loco.c
index 0fdce6d..56f88d1 100755
--- a/arch/arm/mach-mx5/board-mx53_loco.c
+++ b/arch/arm/mach-mx5/board-mx53_loco.c
@@ -650,9 +650,9 @@ static void __init fixup_mxc_board(struct machine_desc *desc, struct tag *tags,
 		gpu_data.reserved_mem_size = gpu_mem;
 
 		/* reserver memory for fb */
-		loco_fb_data[0].res_base = gpu_data.reserved_mem_base
+		loco_fb_data[0].res_base[0] = gpu_data.reserved_mem_base
 					+ gpu_data.reserved_mem_size;
-		loco_fb_data[0].res_size = fb_mem;
+		loco_fb_data[0].res_size[0] = fb_mem;
 	}
 }
 
-- 
1.7.7.4

